app/templates/direct-award/end-search.html

```diff
@@ -45,16 +45,19 @@
       <div class="dmspeak">
         <h2 class="heading-xmedium">Before you end your search</h2>
       </div>
-      <p>You should only continue when you have finished searching for cloud services and are ready to start comparing the services you’ve found.</p>
+      <p>You should only continue when you have finished searching for cloud services.</p>
 
       <br>
-      <p>You can’t edit your search after it has ended.</p>
+      <p>You can not edit your search once it has ended.</p>
```

Comment:
I feel like we generally would opt for 'cannot' or 'can't' before; can you help me understand the difference?

Reply:
Cannot or can not are both correct. The first reads better, so I'll change it.

We're avoiding can't in this instance to reduce the risk of users misreading the negative contraction as can. There's a lot of chat about this in content and accessibility communities at the moment.